dom.bs

```diff
@@ -8642,8 +8642,10 @@ are to:
 <a lt="named attribute"><var>associated attribute's local name</var> attribute</a> or
 associated <a for="/">element</a>'s
 <a lt="named attribute"><var>associated attribute's local name</var> attribute</a> is
-<a lt="attribute is set">set</a>, set <a>tokens</a> to the new <a for=Attr>value</a>,
-<a lt="ordered set parser">parsed</a>.
+<a lt="attribute is set">set</a>, let <var>temporary tokens</var> be the new <a for=Attr>value</a>,
```

Comment:
I would prefer a "run these substeps" if we start creating variables. I think that would be a little easier to read.